### ISSUE TITLE
Implement version of homepage with only one CTA

### ISSUE THREAD

**Issue by hannahkane** [mozilla/teach.webmaker.org#1159]
Blocked by design ticket: #1158

**Comment by hannahkane** [comment]
Good reminder from @HPaulJohnson - should we take this opportunity to install optimizely?

**Comment by hannahkane** [comment]
Also - @KevZawacki had a useful question. Is there a way we can link directly to the pledge? If that were the case, we could link to it in emails, for example.

**Comment by hannahkane** [comment]
Re: Optimizely. Wondering what our homepage traffic needs to be to run statistically valid tests in a reasonable amount of time. Any insight @mmmavis or @adamlofting?

**Comment by adamlofting** [comment]
Another way to estimate this is to look at the existing traffic, and estimate how long a test would take to conclude. 

This is a handy tool for doing that: https://vwo.com/ab-split-test-duration/

The time depends on how much of a difference the alternate version has on your given KPI. If it has a huge impact on your conversion rate, the test will conclude much more quickly.

**Comment by hannahkane** [comment]
Ah, perfect. Thanks, @adamlofting. Can you help me figure out our existing conversion rate (people who visit the homepage and take the pledge) and average number of daily visitors (to the homepage, I assume). I'm not sure how to get those data points.

**Comment by adamlofting** [comment]
Just looking at this quickly between calls, so excuse the brevity.

Teach homepage gets ~500 views / day
The Homepage pledge to teach CTA gets ~40 clicks / day

I don't think we have data in GA about number of successful pledges, but we should file a bug for that as we should be tracking it, and it will be pretty simple to track. If you have data for number of sign-ups to hand, you could divide this by number of days to get an average. I'm guessing this is in BSD, but don't have time to look right now.

**Comment by hannahkane** [comment]
Thanks, @adamlofting. It is in BSD. @mmmavis - do you have time to grab it?

**Comment by adamlofting** [comment]
I just had a look, and it's about 6 / day.

**Comment by adamlofting** [comment]
Plugging some numbers into the calculator:

* If your test doubled the conversion rate (100% increase), you could get significant results in 5 days
* If you made a 50% improvement, in 21 days
* But a 20% improvement would need 132 days to get significant results

So the key is to test bold, dramatic variations which are likely to have the most impact.

**Comment by hannahkane** [comment]
Super. Thanks, @adamlofting. I think our first test—one CTA versus 3 CTAs—is a pretty dramatic variation. Excited to test this and see.

**Comment by jgmac1106** [comment]
A low priority question, really just curious:

 How do you know your conversion rate is due to the changes in the CTA? For example what if testing happens the same time as an email blast about Maker Party, Clubs or Thimble relaunch? Is the link on these emails to teach.mozilla.org big and at the top? Small and at the bottom?

If you need your conversions to move to 12 out of 40 or 24 out of 80 (if unique visitors were to double) could any of these other events lead to six more people pledging rather than moving to one CTA versus three?

**Comment by adamlofting** [comment]
@jgmac1106 it's a good question to ask.

The A/B testing process works by running a controlled experiment. So half of the traffic are randomly allocated to see the existing version (the control), and half to see the new variation over the same period of time.

So if we sent an email blast during the test period, 50% of the people responding to the email will see version A, and 50% will see version B. For direct visitors to the website, the split is the same, and so on.

It's important that the test should run for long enough and have enough visitors that it represents your target audience as a whole. Running a test just on a weekend for example, might optimize for the wrong audience.

Then you analyse the difference in results for version A and B to see if it's statistically significant. 

Without wanting to distract, there are [layers of complexity](http://9slides.com/Talks/Things-they-don-t-tell-you-about-conversion-rates) under this. You can increase conversion rate while decreasing 'quality' of responses for example. But as a starting point, A/B testing for conversion rate is a good thing to work on.

**Comment by jgmac1106** [comment]
Wow thx! Terminating thread hijacking ::poof::

**Comment by hannahkane** [comment]
No longer blocked on design.

**Comment by mmmavis** [comment]
A/B testings between 1 CTA vs 3 CTAs have started! :dancer:

**Comment by hannahkane** [comment]
woohoo!
